README.md

```diff
@@ -1,7 +1,12 @@
 # Vorlage-Latex
-Dies ist eine LateX Vorlage für Projektarbeiten und Belege.
+Dies ist eine LateX Vorlage für Projekt-, Beleg-, Bachelor- und Diplomarbeiten an der Staatlichen Studienakademie Glauchau (BA Glauchau).
 
-Unterhaltung und Besprechung:
+Die Vorlage hält sich an die [Hinweise zur Anfertigung wissenschaftlicher Arbeiten (HAWA)](https://www.ba-glauchau.de/fileadmin/glauchau/waehrend-des-studium/dokumente/pruefungen/4BA-F.207_Hinweise_zur_Anfertigung_wissenschaftlicher_Arbeiten.pdf).
+
+Die oben genannten unterstützten Versionen sind jeweils für 1-4 Beteiligte problemlos verwendbar.
+
+
+**Unterhaltung und Besprechung:**
```

Comment:
ist das gewollt, dass das keine liste ist? (mit * in markdown)

Reply:
Ist gewollt.